### DISCUSSION TITLE
DB Browser for SQLite

### DISCUSSION THREAD

**Discussion by mikevio** [sqlitebrowser/sqlitebrowser#3548] [Q&A]
Sorry if this sounds ridiculous, but I am very new to the community.  I cannot get the query tab label to expand! I only see a few letters and "..."  Am I missing something?  I tried dragging, closing other windows, etc.  Thanks!  Please see the attached screenshot.
[DB Browse SQLLite.pdf
[DB Browse SQLLite.pdf](https://github.com/sqlitebrowser/sqlitebrowser/files/13880015/DB.Browse.SQLLite.pdf)
](https://github.com/sqlitebrowser/sqlitebrowser/files/13880010/DB.Browse.SQLLite.pdf)

**Comment by chrisjlocke** [comment]
I would guess this is a font issue.  While the font size can be changed, I don't think the system font can - what DB4S uses to draw tabs, et al.
Can you try the nightly version to confirm that has the same issue?  I'm guessing it does, but its good to check.
https://nightlies.sqlitebrowser.org/latest/
What versoin of DB4S are you using, and what macOS are you using?

**Reply by mikevio** [reply]
Thank you sooo much for the response! Same issue with the nightly version but worth a shot.

OS is Sonoma Version 14.2.1 (23C71)

DB Browser for SQLite Version 3.12.2

Built for x86_64-little_endian-lp64, running on x86_64
Qt Version 5.15.2
SQLCipher Version 4.4.3 community (based on SQLite 3.34.1).

Looking into font issue now.
All the best!!!

**Comment by richb-hanover** [comment]
I also see the same thing on macOS 12.7.2. It has always been present as far as I can remember - at least two years? (I always figured it simply displayed "SQL query" or some anodyne thing, so I didn't worry about it.)  

@mikevio - what would you _like_ it to say, if DB4S could display more?

Here's what the nightly build looks like on my Mac. Note the multiple tabs showing "S...": Thanks!

<img width="1156" alt="image" src="https://github.com/sqlitebrowser/sqlitebrowser/assets/1094930/8836fd95-fd7a-4070-bcee-d590849261bd">
